src/write_gcm.c

```diff
@@ -3244,26 +3335,30 @@ static int wg_json_CreateTimeSeries(
     wg_json_string(jc, "metric");
     wg_json_Metric(jc, head);
 
+    char *metric_value_type_string = wg_metric_value_type_to_string(
```

Comment:
How do we handle the `NULL` that you return in case of an error?

Reply:
Ack via `UNKNOWN` equivalent.